### PR TITLE
Feat: Landing Page

### PR DESCRIPTION
**Pull request by Sandesh-Ghimire**
## Issue

- [ ] Fixes - #5 
- [x] Partially fixes - #5

### CODE REVIEW

**Comment by Sandesh-Ghimire** [comment]
Team: i have tried to create landing page sample from the link that i have provided in group discussion. It is incomplete , if anyone want to add some cool stuff, you can do so.

File: src/components/Landing/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import { Midsection } from "./why";
```
Comment (nirajgiriXD) [inline src/components/Landing/index.tsx:2]:
```suggestion
/**
 * External dependencies.
 */
import React from "react";

/**
 * Internal dependencies.
 */
import { Midsection } from "./why";

```

File: src/components/Landing/index.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+export  function Landingpage()
```
Comment (nirajgiriXD) [inline src/components/Landing/index.tsx:3]: Use arrow function creating components.
```
- export  function Landingpage()
```
```
+ const LandingPage = () => {}
+ export default LandingPage;
```

**Review by nirajgiriXD — CHANGES_REQUESTED** [review]
## General Feedback:

- Ensure that the coding pattern matches to those already implemented in the project, even for sections that are incomplete
- Download the images and put them in the `assets/img` folder
- Use arrow function for creating components
- Add dependency comments at top

**Pull request closed without merging**